### PR TITLE
Update dial-pad-configuration.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5686
Added NOTE.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/dial-pad-configuration.md [hunk 1]
```diff
@@ -29,6 +29,9 @@ In the Teams client, the dial pad enables users to access Public Switched Teleph
 - User is homed online and not in Skype for Business on premises
 - User has Teams Calling Policy enabled
 
+> [!NOTE]
+> There is now a restriction that only allows the dial pad to appear if the user is set to Teams only. Teams does not support the usage of enterprise voice (MS Hosted phone Service) on two different applications, such as Skype for Business Online and Teams at the same time.
```
Comment (JohanFreelancer9) [inline Teams/dial-pad-configuration.md:33]:
```suggestion
> There is now a restriction that only allows the dial pad to appear if the user is set to Teams only. Teams does not support the usage of Enterprise Voice (Microsoft-hosted phone service) on two different applications, such as Skype for Business Online and Teams at the same time.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by CarolynRowe** [comment]
@scanum:  Did we check with PMs on this verbiage?  If not, I recommend we do so.  Let me know and I can follow up.

**Comment by CarolynRowe** [comment]
The wording in this PR could be misleading.  I will follow up with the PMs on the correct wording.  Closing this PR for now.

**Pull request closed without merging**